docs(bil): faq and how to update (#3186)
scaleway/docs-content: faq/billing.mdx

## How can I obtain a SEPA Mandate?

Refer to the [How to add a payment method](/console/billing/how-to/add-payment-method#how-to-add-a-sepa-mandate) documentation page for a complete guide on how to add a SEPA mandate.

> ## My payment verification code expired. What should I do?

Author: ```suggestion
## My credit card validation code expired. What should I do?
```

> If your verification code expires, please [contact our support team](https://console.scaleway.com/support/tickets/create) to request a new magic code.

Reviewer: Maybe we should still to either calling it a verification code or a magic code? I don't know which one is actually used in the console? Below we also mention "transaction code" - it could be confusing to use different names for the same thing?

Author: Good point. I'll update it to validation code, to match the console. Thanks, Ro!

> If your verification code expires, please [contact our support team](https://console.scaleway.com/support/tickets/create) to request a new magic code.

Author: ```suggestion
If your validation code expires, please [contact our support team](https://console.scaleway.com/support/tickets/create) to request a new code.
```

## I tried to add a credit card but received the message 'Card already registered'

> A credit card can only be registered once across all Organizations If you receive this message, make sure you do not have the same card already registered to another Scaleway account.

Reviewer: ```suggestion
A credit card can only be registered once across all Organizations. If you receive this message, make sure you do not have the same card already registered to another Scaleway account.
```

If you only have one payment method and need to use it on more than one account, [contact our support team](https://console.scaleway.com/support/tickets/create) to request an individual verification.

## I've added a physical credit card, but it was declined. Why?

If your credit card complies with the payment requirements but still was declined, please [contact our support team](https://console.scaleway.com/support/tickets/create) for assistance.

> ## I added a credit card but didn’t receive a magic code. What do I do?

Reviewer 2: Ici on peut peut etre expliquer au client qu'il peut contacter le support pour recevoir un nouveau magic code

Author: Done. Merci!

It usually takes up to 24 hours to receive a magic code.

> If after that period you still don't receive it, contact your bank or credit card provider to request the transaction code.

Author: ```suggestion
If after that period you still don't receive it, contact your bank or credit card provider to request the transaction code or [contact our support team](https://console.scaleway.com/support/tickets/create) to request a new code.
```

## How long does it take to receive a magic code?

Usually, the magic code (transaction code) shows up in your bank statement in up to 24 hours.

## Why can’t I delete my credit card?

> It might not be possible to remove your credit card from your account if you have one or more outstanding invoices. If that is the case, make sure you have paid any issued invoice(s) and have no pending consuming over the current calendar month, and retry.

Reviewer: ```suggestion
It might not be possible to remove your credit card from your account if you have one or more outstanding invoices. If that is the case, make sure you have paid any issued invoice(s) and have no consumption in the current calendar month, and retry.
```

> You might also be unable to delete a credit card if it is your only registered payment method. In this case, you can add a new credit card, select it as a default payment method, and then delete the previous credit card from your account.

Reviewer 2: Je pense que ce pragraphe devrait etre placé en premier. 
Aussi une carte de credit ne peut pas etre supprimé meme si un SEPA est enregistré et valide. Il faut toujours une CB active sur le compte.

Author: Merci, Julie. Je maj la reponse.

If your case does not match the situations above and you still cannot remove your credit card, [contact our support team](https://console.scaleway.com/support/tickets/create) for a manual verification.

## How can I change the country or region registered to my account?

It is not possible to edit your country and region directly from the Scaleway console. Please [contact our support team](https://console.scaleway.com/support/tickets/create) if you need to change this information.

> Be prepared to inform your new address, including country, region, city, postal code, and street address. Our team will then proceed to update your account details.

Reviewer: ```suggestion
Be prepared to inform us of your your new address, including country, region, city, postal code, and street address. Our team will then proceed to update your account details.
```

> ## I was billed for resources that are deactivated or powered off

Reviewer: ```suggestion
## Why have I been billed for resources that are deactivated or powered off?
```

If you were billed for resources that have been correctly stopped, please [contact our support team](https://console.scaleway.com/support/tickets/create), providing the name of the product and resource, and the date of deactivation.

## Can I use a virtual or prepaid card as a payment method?

No. Virtual or prepaid cards are **not** accepted as payment methods. When using a credit card, ensure you are using a physical card, protected by 3D secure.

## I deleted my account, but still received an invoice. Why?

> Scaleway invoices are calculated at the end of each month, so you may receive a final invoice the month after closing your account. That refers to your outstanding consumption for your final month of activity.

Reviewer: ```suggestion
Scaleway invoices are calculated at the end of each month, so you may receive a final invoice the month after closing your account. That corresponds to your outstanding consumption for your final month of activity.
```

## I tried to add a credit card but received the message 'Card already registered'

You can only use one credit card per Organization. If you receive this message, make sure you do not have the same card already registered to another Scaleway account.

If you only have one payment method and need to use it on more than one account, [contact our support team](https://console.scaleway.com/support/tickets/create) to request an individual verification.

## I've added a physical credit card, but it was declined. Why?

If your credit card complies with the payment requirements but still was declined, please [contact our support team](https://console.scaleway.com/support/tickets/create) for assistance.

## I added a credit card but didn’t receive a magic code. What do I do?

It usually takes up to 24 hours to receive a magic code.

If after that period you still don't receive it, contact your bank or credit card provider to request the transaction code.

## How long does it take to receive a magic code?

Usually, the magic code (transaction code) shows up in your bank statement in up to 24 hours.

## Why can’t I delete my credit card?

> It might not be possible to remove your credit card from your account if you have one or more outstanding invoices. If that is the case, make sure to pay the open invoice(s) and retry.

Author: ```suggestion
It might not be possible to remove your credit card from your account if you have one or more outstanding invoices. If that is the case, make sure you have paid any issued invoice(s) and have no pending consuming over the current calendar month, and retry.
```
